### ISSUE TITLE
Errors with Post Properties Meta Box

### ISSUE THREAD

**Issue by miklb** [dshanske/indieweb-post-kinds#119]
Currently on master (2.7.0) and when editing [this post](https://miklb.com/2017/11/christian-heilmann-web-truths-the-web-is-better-than-any-other-platform-as-it-is-backwards-compatible-and-fault-tolerant/) where I manually added the author name due to lack of mf2 properties, I get this stack trace in debug.log

```
[24-Nov-2017 18:21:02 UTC] PHP Warning:  Illegal string offset 'name' in /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/tabs/tab-author.php on line 13
[24-Nov-2017 18:21:02 UTC] PHP Stack trace:
[24-Nov-2017 18:21:02 UTC] PHP   1. {main}() /Users/miklb/Dev/miklb.dev/wp-admin/post.php:0
[24-Nov-2017 18:21:02 UTC] PHP   2. include() /Users/miklb/Dev/miklb.dev/wp-admin/post.php:174
[24-Nov-2017 18:21:02 UTC] PHP   3. do_meta_boxes() /Users/miklb/Dev/miklb.dev/wp-admin/edit-form-advanced.php:714
[24-Nov-2017 18:21:02 UTC] PHP   4. Kind_Tabmeta::display_metabox() /Users/miklb/Dev/miklb.dev/wp-admin/includes/template.php:1063
[24-Nov-2017 18:21:02 UTC] PHP   5. include_once() /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/class-kind-tabmeta.php:200
[24-Nov-2017 18:21:02 UTC] PHP   6. require_once() /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/tabs/tab-navigation.php:12
[24-Nov-2017 18:21:02 UTC] PHP Fatal error:  Uncaught Error: Only variables can be passed by reference in /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/tabs/tab-author.php:13
Stack trace:
#0 /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/tabs/tab-navigation.php(12): require_once()
#1 /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/class-kind-tabmeta.php(200): include_once('/Users/miklb/De...')
#2 /Users/miklb/Dev/miklb.dev/wp-admin/includes/template.php(1063): Kind_Tabmeta::display_metabox(Object(WP_Post), Array)
#3 /Users/miklb/Dev/miklb.dev/wp-admin/edit-form-advanced.php(714): do_meta_boxes(Object(WP_Screen), 'normal', Object(WP_Post))
#4 /Users/miklb/Dev/miklb.dev/wp-admin/post.php(174): include('/Users/miklb/De...')
#5 {main}
  thrown in /Users/miklb/Dev/miklb.dev/wp-content/plugins/indieweb-post-kinds/includes/tabs/tab-author.php on line 13
```

https://christianheilmann.com/2017/11/18/web-truths-the-web-is-better-than-any-other-platform-as-it-is-backwards-compatible-and-fault-tolerant/ is the Url retrieved in the meta box if you would like to test.

Let me know what else I can provide.

```
indieweb-post-kinds> php --version
PHP 7.1.11 (cli) (built: Oct 27 2017 11:00:03) ( NTS )
```

**Comment by dshanske** [comment]
Believe this is now fixed.

**Comment by miklb** [comment]
Not at least in my test environment, same fatal error and stack trace v2.7.1 and WordPress 4.9

**Comment by miklb** [comment]
Fixed in 7d4fe99fdcf59933874d6a6ffb5cc8bbba749d8d
